doc/whats_new/v1.3.rst

```diff
@@ -51,11 +51,24 @@ Changes impacting all modules
   - :class:`sklearn.preprocessing.RobustScaler`
   - :class:`sklearn.preprocessing.StandardScaler`
   - :class:`sklearn.feature_extraction.text.TfidfTransformer`
+  - :class:`sklearn.random_projection.GaussianRandomProjection`
+  - :class:`sklearn.random_projection.SparseRandomProjection`
+  - :class:`sklearn.feature_selection.GenericUnivariateSelect`
+  - :class:`sklearn.feature_selection.RFE`
+  - :class:`sklearn.feature_selection.RFECV`
+  - :class:`sklearn.feature_selection.SelectFdr`
+  - :class:`sklearn.feature_selection.SelectFpr`
+  - :class:`sklearn.feature_selection.SelectFromModel`
+  - :class:`sklearn.feature_selection.SelectFwe`
+  - :class:`sklearn.feature_selection.SelectKBest`
+  - :class:`sklearn.feature_selection.SelectPercentile`
+  - :class:`sklearn.feature_selection.SequentialFeatureSelector`
+  - :class:`sklearn.feature_selection.VarianceThreshold`
```

Comment:
```suggestions
  - :class:`random_projection.GaussianRandomProjection`
  - :class:`random_projection.SparseRandomProjection`
  - :class:`feature_selection.GenericUnivariateSelect`
  - :class:`feature_selection.RFE`
  - :class:`feature_selection.RFECV`
  - :class:`feature_selection.SelectFdr`
  - :class:`feature_selection.SelectFpr`
  - :class:`feature_selection.SelectFromModel`
  - :class:`feature_selection.SelectFwe`
  - :class:`feature_selection.SelectKBest`
  - :class:`feature_selection.SelectPercentile`
  - :class:`feature_selection.SequentialFeatureSelector`
  - :class:`feature_selection.VarianceThreshold`
```

Reply:
It could also be worth to reorder in alphabetic order.

Reply:
Sure. Thanks

Reply:
Just curious, ordered alphabetically by class name or module name?